packages/jest-emotion/test/react-enzyme.test.js

```diff
@@ -0,0 +1,17 @@
+import 'test-utils/legacy-env'
+/** @jsx jsx */
+import * as enzyme from 'enzyme'
+import { css, jsx } from '@emotion/core'
+import { createSerializer as createEnzymeSerializer } from 'enzyme-to-json'
+import { createSerializer } from 'jest-emotion'
+
+expect.addSnapshotSerializer(createEnzymeSerializer())
+expect.addSnapshotSerializer(createSerializer())
+
+test('enzyme mount test', () => {
+  const Greeting = ({ children }) => (
+    <div css={css({ backgroundColor: 'red' })}>{children}</div>
```

Comment:
Not related to this PR but you don't need the css call here, you can pass the style object directly to the css prop.

Reply:
Ah yep, thanks for updating this